data/protected-disease-gene.tsv

```diff
@@ -0,0 +1,7 @@
+omim_id	mondo_id	mondo_label	orcid	comment
+OMIM:619151	MONDO:0030894	"AMED syndrome, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
```

Comment:
@joeflack4 can you help me with a sanity check? I don't see two genes listed on the OMIM web site for the entries in this file now. Can you check at least one of these OMIMs and confirm that there is now only one gene association in the?

Reply:
@twhetzel All of them only have 1 association in `morbidmap.txt`. I didn't add these here so I can't be sure they ever did have 2. They say "digenic" on them which may be why we gave them special attention. Perhaps the fact that they only have 1 association is not surprising then; it's consistent with Sabrina's / ours decision that these should be protected / added as disease defining.

Reply:
Thanks, the entries in the protected file do have 2 gene associations in Mondo though.